chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/RoomSubscriptionGroup.kt

```diff
@@ -38,19 +37,19 @@ internal class RoomSubscriptionGroup(
     )
 
     private val cursorsSubscription = cursorService.subscribeForRoom(
-            roomId,
-            consumer = cursorConsumer
+            roomId
+//            consumer = cursorConsumer
     )
 
     override fun connect(): Subscription {
         roomSubscription.await()
-        cursorsSubscription.await()
+//        cursorsSubscription.await()
```

Comment:
cursorsSubscription await()/unsubsribe() lines do not need to be commented out, do they?

Reply:
After removing the RoomService, I found that the RoomSubscriptionGroup was no longer useful too - would you prefer I put it back?

Reply:
I think it's fine to remove it as well. We have the reference in v1's code and we can use it when implementing similar logic in v2.

Reply:
👍 that makes sense - thank you for clarifying!